crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -521,17 +527,18 @@ impl<Wg: WireGuard, I: UpnpEpCommands, E: Backoff> State<Wg, I, E> {
                 }
             }
             Err(e) => {
-                if let Error::NoIGDGateway = e {
-                    telio_log_info!("Failed to check route: route does not exist")
-                } else {
-                    telio_log_info!("Error: {}", e);
+                telio_log_info!("Error validating endpoint: {}", e);
 
-                    telio_log_info!("Deleting an old Upnp endpoint");
-                    let detete = self.igd_gw.delete_endpoint_routes(
+                telio_log_info!("Dropping Upnp endpoint");
+                if let Err(e) = self
+                    .igd_gw
+                    .delete_endpoint_routes(
                         self.proxy_port_mapping.external,
                         self.wg_port_mapping.external,
-                    );
-                    let _ = detete.await;
+                    )
+                    .await
+                {
+                    telio_log_info!("Error dropping Upnp endpoint: {}", e);
```

Comment:
This looks like it should be an error log, not info.

Reply:
This error could be simply the mappings were already dropped by the UPnP device, as the comment above, I don't consider it worth to be logged as a error, as the runtime is not affected and the mappings won't exist anyway.